Userland/Libraries/LibAudio/PlaybackStreamSerenity.cpp

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2023, Gregory Bertilson <zaggy1024@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "PlaybackStreamSerenity.h"
+
+#include <LibCore/ThreadedPromise.h>
+
+namespace Audio {
+
+ErrorOr<NonnullRefPtr<PlaybackStream>> PlaybackStreamSerenity::create(OutputState initial_state, u32 sample_rate, u8 channels, u32 target_latency_ms, AudioDataRequestCallback&& data_request_callback)
+{
+    VERIFY(data_request_callback);
+    auto connection = TRY(ConnectionToServer::try_create());
+    if (auto result = connection->try_set_self_sample_rate(sample_rate); result.is_error())
+        return Error::from_string_literal("Failed to set sample rate");
+    (void)channels, (void)target_latency_ms;
```

Comment:
not super nice to do this with a comma inbetween. Would prefer either sep lines, or to slap [[maybe_unused]] in the function definition prototype thing on line 13.